charter.md

```diff
@@ -14,22 +14,26 @@ proxy. Moreover, it would not introduce potentially redundant or unnecessary rec
 an established request/response semantic that can set up and configure flows for different services.
 
 The primary goal of this working group is to develop mechanism(s) that allow configuring and concurrently running
-multiple proxied stream- and datagram-based flows inside an HTTPS connection. These mechanism(s) are collectively called
+multiple proxied stream- and datagram-based flows inside an HTTP connection. These mechanism(s) are collectively called
 MASQUE. The group will specify HTTP and/or HTTP/3 extensions to enable this functionality.
 
-The group will focus on a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying.
-Server-initiated services are out of scope. The working group will first deliver a protocol solution for UDP CONNECT and
-a requirements document for IP proxying. Once both are complete, the working group will focus on a protocol solution for
-IP proxying.
-
-The working group will consider fallback to versions of HTTPS that operate over TCP as a mitigation to UDP or HTTP/3
-blocking. Moreover, the working group will consider implications of tunneling protocols with congestion control and loss
-recovery over MASQUE, and may issue recommendations accordingly. New congestion control and loss recovery algorithms are
-out of scope.
-
-Multicast support is out of scope. However, the group may specify extension points that would enable future work on
-multicast. Specifying proxy server discovery mechanisms is also out of scope. However, the group may consider features,
-such as proxy server identifiers, that aid future discovery mechanisms.
+The group will focus on extensions for a limited set of client-initiated services: (1) UDP CONNECT and (2) IP proxying.
+Server-initiated services are out of scope.
+
+Exercising the extension points defined by CONNECT-UDP and CONNECT-IP helps make it easier to support new use cases or
+accommodate changes in the environment in which these protocols are deployed. As such, the working group will consider
+adopting an extension to these documents when it provides missing functionality or otherwise eases deployability for
+use cases, where there are multiple implementation and/or deployment proponents. The initial set of extensions will be
+in support of UDP listening and QUIC-aware proxying. Additional extensions that help enable CONNECT-UDP or CONNECT-IP
+services to coordinate or work well in the ecosystem of other services may be considered. Extensions to HTTP Datagrams
```

Comment:
These two sentences seem to overlap a fair bit.

> As such, the working group will consider adopting an extension to these documents when it provides missing functionality or otherwise eases deployability for use cases, where there are multiple implementation and/or deployment proponents.

And

> Additional extensions that help enable CONNECT-UDP or CONNECT-IP services to coordinate or work well in the ecosystem of other services may be considered. 

Can this be simplified?

Reply:
Thanks, reworked that paragraph and it is now much more concise